src/main/java/ToDo.java

```diff
@@ -0,0 +1,11 @@
+public class ToDo extends Task{
+
+    public ToDo(String description) {
+        super(description);
+    }
+
+    @Override
```

Comment:
I like that you added the Override tag so that it's immediately clear that this method is being overridden